testscommon/preprocessorMock.go

```diff
@@ -0,0 +1,163 @@
+package testscommon
```

Comment:
Why duplicate this with the one from `mock/preprocessorMock.go` ?

Reply:
We should've had only one in testscommon in a separate package to avoid cyclic imports instead of keeping it in another package.

But it can stay as it is for now.